x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/set_alert_tags_route.ts

```diff
@@ -100,7 +100,7 @@ export const setAlertTagsRoute = (router: SecuritySolutionPluginRouter) => {
         try {
           const body = await esClient.updateByQuery({
             index: `${DEFAULT_ALERTS_INDEX}-${spaceId}`,
-            refresh: false,
+            refresh: true,
```

Comment:
To fix similar issue with assignees updates in alerts table, I made util methods `waitForAssigneeToAppearInTable` and `waitForAssigneeToDisappearInTable`:

https://github.com/elastic/kibana/pull/167079/files#diff-748d7f4681b7ad904109c3c6537257e8414060674c9856aa27e52ee0994369a6R23


My tests were based on `alert_tags.cy.ts`, so it might also work for tags. But not 100% sure what approach is better here :-)